en/docs/security-processes/cloud-security-process.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud Security Process
+category: security-processes
+published: 5th May 2023
+version: 1.0
+---
+
+# Cloud Security Process
+
+## Overview
+We have incorporated DevSecOps methodologies into our Secure Software Development Lifecycle, employing cloud-native security best practices. This approach has fostered a collective responsibility for security throughout our teams.
+
+This document is organized into sections that outline the secure design patterns and security measures we have put in place to safeguard against major threats. These threats include internal attacks, software supply chain attacks, employee account takeovers, service attacks, platform attacks, and more.
+
+We have closely aligned our process with "*The Six Pillars of DevSecOps: Pragmatic Implementation*" by Cloud Security Alliance[^1].
+
+
+## Secure Software Development Lifecycle  
+The Secure Software Development Lifecycle (SSDLC) is an approach that embeds security within every stage of the software development process, from inception to deployment and maintenance. In this methodology, security is not treated as an afterthought but as a fundamental aspect throughout the entire SDLC.
+
+We recognize the importance of adopting secure development practices to reduce the risk of vulnerabilities and cyberattacks. To accomplish this, we integrate security as a crucial component in each phase of our SDLC. This ensures that security considerations are incorporated into the design, development, testing, deployment, and maintenance stages of their software development process.
+
+By integrating security into each stage of our SDLC, we strive to guarantee that software products and solutions are designed with security as a priority from the beginning. This approach helps minimize the chances of vulnerabilities and other security issues that may be identified later in the development process or after deployment. The ultimate outcome is a more secure software product capable of mitigating potential security risks and defending against cyber threats.
+
+![Cloud Software Development Process - stages](../assets/images/cloud-security-process/cloud-software-development-process-stages.png)
```

Comment:
Shall we add this one later. Need to review it better and make it a bit more graphical. Let's include it to the next version.